src/Repository/UserRepository.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+
+namespace Marketredesign\MrdAuth0Laravel\Repository;
+
+use Auth0\SDK\API\Management;
+use GuzzleHttp\Exception\RequestException;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Cache;
+
+class UserRepository implements \Marketredesign\MrdAuth0Laravel\Contracts\UserRepository
+{
+    private const CACHE_TTL = 1800;
+
+    private $mgmtApi;
+
+    /**
+     * UserRepository constructor.
+     * @param Management $management
+     */
+    public function __construct(Management $management)
+    {
+        $this->mgmtApi = $management;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function get($id): ?object
+    {
+        if ($id == null) {
+            return null;
+        }
+
+        $cacheKey = 'auth0-users-get' . $id;
```

Comment:
```suggestion
        $cacheKey = 'auth0-users-get-' . $id;
```
Not very important, but I think it'd be neater, keeping all sections separated with a `-`.